Update hello dojo, Graphql documentation

gianalarcon opened the pull request:
I prepared a quick documentation of Graphql inside hello dojo section. I covered
- Query structure (component).
- Subscribing to entity.
Along the example provided

At the same time, I created a dedicated section for Graphql inside torii section

This is working with dojo stable version -v0.2.2

Partially solves #44

gianalarcon commented:
@ponderingdemocritus, I need your review and feedback. If everything looks ok, I will wait until v0.3 is released to complete the full documentation.

To-Do:
- Add more query examples.
- Add subscription by ID examples.
- Rename "component" to "model".

ponderingdemocritus commented:
@gianalarcon I have been making changes on 0.3.0 branch, could you PR this into that branch? 

There will be conflicts in hello-dojo as I updated it to models from components.

gianalarcon commented:
@ponderingdemocritus I changed my PR into 0.3.0 branch, and I partially updated my PR according to  0.3.0 branch. I want to add and updated more documentation, but currently `sozo init` still create a project v0.2.2 in hello-dojo.dm
`dojo = { git = "https://github.com/dojoengine/dojo", tag="v0.2.2"}`
Or you can share with me the codebase of project sample for hello-dojo.
